syne_tune/optimizer/schedulers/regularized_evolution.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
```

Comment:
General comment: This is in fact a searcher, feeding into FIFOScheduler.

Doing it that way supports points_to_evaluate, does random seed management in the way the other schedulers do, and is easy to link into the CLI. Also, our users understand this is just another way of FIFO scheduling with a clever selection of the config.

Reply:
A simple example for how a searcher looks like, is RandomSearcher. If you really want to have very little work, you could create an internal instance of RandomSearcher to do the "points_to_evaluate" and random decisions for you. But you can also look at how RandomSearcher does it. This includes important things, like checking whether the search space is exhausted. Features we do not want to lose.

Reply:
You can also check KernelDensityEstimator, which inherits from SearcherWithRandomSeed, so the random seed stuff is all there.